uint32 cameraId 				#index to the camera whose module wants image (see Camera Indexes)
uint32 cameraParameterId 	#Id of the camera parameter (see Camera parameters)
uint32 newValue 				#Parameterâ€™s new value.
---
byte isSet
